optaplanner-examples/pom.xml

```diff
@@ -59,15 +59,11 @@
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>exec-maven-plugin</artifactId>
           <configuration>
-            <executable>java</executable>
```

Comment:
By deleting this line doesn't the configuration apply to "mvn exec:exec" instead of "mvn exec:java"?
However, it seems that we need to be using "mvn exec:exec" indeed, according to the jira below, so this change looks right.
  https://issues.jboss.org/browse/PLANNER-287
We need to be sure that what is use portable to windlows' fragile command line length limitation too.
I'll need to check if the docs never reference the wrong mvn form.